packages/ti_google_threat_intelligence/kibana/dashboard/ti_google_threat_intelligence-0b0fb6b4-d250-4e31-a56a-bb872e4c7c4a.json

```
@@ -0,0 +1,953 @@
+                            "alias": null,
+                            "disabled": false,
+                            "field": "gti.threat.indicator.attributes.last_analysis_stats.malicious",
+                            "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[1].meta.index",
+                            "key": "gti.threat.indicator.attributes.last_analysis_stats.malicious",
+                            "negate": false,
+                            "params": {
+                                "gte": "1"
+                            },
+                            "type": "range"
+                        },
+                        "query": {
+                            "range": {
+                                "gti.threat.indicator.attributes.last_analysis_stats.malicious": {
+                                    "gte": "1"
+                                }
+                            }
+                        }
+                    }
+                ],
+                "query": {
+                    "language": "kuery",
+                    "query": ""
+                }
+            }
+        },
+        "optionsJSON": {
+            "hidePanelTitles": false,
+            "syncColors": false,
+            "syncCursor": true,
+            "syncTooltips": false,
+            "useMargins": true
+        },
+        "panelsJSON": [
+            {
+                "embeddableConfig": {
+                    "enhancements": {
+                        "dynamicActions": {
+                            "events": []
+                        }
+                    },
+                    "savedVis": {
+                        "data": {
+                            "aggs": [],
+                            "searchSource": {
+                                "filter": [],
+                                "query": {
+                                    "language": "kuery",
+                                    "query": ""
+                                }
+                            }
+                        },
+                        "description": "",
+                        "id": "",
+                        "params": {
+                            "fontSize": 12,
+                            "markdown": "[Threat Intelligence](#/dashboard/ti_google_threat_intelligence-9e8de699-a623-4a1b-9f63-7d641116f531), **Adversary Intelligence**, [Threat Feed Overview](#/dashboard/ti_google_threat_intelligence-95187e5c-b4a2-45ad-b6a4-d6ce68e1f43e)\n\nThis dashboard provides adversary intelligence insights into threat intelligence events detected in the user's environment by Google Threat Intelligence.\n\nIt categorizes detected threats into malware families, collections, campaigns, reports, software toolkits, threat actors, and vulnerabilities, using saved searches and uses metrics for count of unique adversaries.\n\n[**Integrations Page**](/app/integrations/detail/ti_google_threat_intelligence/overview)",
```

I'm not convince that this need commas between the targets.
```suggestion
                            "markdown": "[Threat Intelligence](#/dashboard/ti_google_threat_intelligence-9e8de699-a623-4a1b-9f63-7d641116f531) **Adversary Intelligence** [Threat Feed Overview](#/dashboard/ti_google_threat_intelligence-95187e5c-b4a2-45ad-b6a4-d6ce68e1f43e)\n\nThis dashboard provides adversary intelligence insights into threat intelligence events detected in the user's environment by Google Threat Intelligence.\n\nIt categorizes detected threats into malware families, collections, campaigns, reports, software toolkits, threat actors, and vulnerabilities, using saved searches and uses metrics for count of unique adversaries.\n\n[**Integrations Page**](/app/integrations/detail/ti_google_threat_intelligence/overview)",
```